examples/annotator_2d.py

```diff
@@ -25,7 +25,7 @@ def livecell_annotator(use_finetuned_model):
         embedding_path = os.path.join(EMBEDDING_CACHE, "embeddings-livecell.zarr")
         model_type = "vit_h"
 
-    annotator_2d(image, embedding_path, show_embeddings=False, model_type=model_type)
```

Comment:
1. If the `show_embedding` kwarg is removed, does that mean there is documentation that needs to be changed? I don't remember if there are docs pages specifically about displaying a visualization of the embedding.

2. Does this mean there is now a separate function for displaying embeddings, or is it removed altogether? I don't have an opinion here (it's not a feature I've ever really used), just checking what the status is

Reply:
In general: I decided to remove this functionality to reduce the complexity because I also don't think it's particularly useful on the user end.

Re 1.: I don't think there was any documentation on this that would need to be updated.

Re 2.: There is no convenience function to display the emebddings any more. There is still the implementation for it in https://github.com/computational-cell-analytics/micro-sam/blob/master/micro_sam/visualization.py. For now I would leave it at that. I think displaying the embeddings is unnecessary for plugin users, so we don't need to expose this functionality to them. On the other hand it can be quite helpful for debugging and method development, so I don't want to fully remove the functionality.

Reply:
Re updating the documentation in general: of course this will need quite some updates to the documentation, but we should work on that separately (there's some more changes necessary beforehand, I will create follow-up issues after merging this one.)

Reply:
> In general: I decided to remove this functionality to reduce the complexity because I also don't think it's particularly useful on the user end.

Agree, I support your decision here